scripts/households.py

```diff
@@ -63,11 +63,11 @@ def FOCs(b_sp1, n_s, *args):
 
 
 
-def get_c(r, w, n_s, b_s, b_sp1):
+def get_c(r, w, n_s, b_s, b_sp1, BQ, N_tilte):
     '''
     Use the budget constraint to solve for consumption
     '''
-    c = w * n_s + (1 + r) * b_s - b_sp1
+    c = w * n_s + (1 + r) * b_s + (BQ / N_tilde) - b_sp1
```

Comment:
You won't need `N_tilde` here.  See equation 8.29.  All the variables in this model will be the "hat" variables - these are the stationary versions of each variable.